scripts/run_backend_tests.py

```diff
@@ -439,7 +439,10 @@ def main(args=None):
             servers.managed_cloud_datastore_emulator(clear_datastore=True))
         stack.enter_context(servers.managed_redis_server())
         if parsed_args.test_target:
-            if parsed_args.test_target.endswith('_test'):
+            if (
+                    parsed_args.test_target.endswith('_test')
+                    or '_test.' in parsed_args.test_target
```

Comment:
Thanks for the PR @IamEzio, can you explain why we need both these checks? (maybe leave a code comment for future readers as well.)

Also, can you please add a unit test to ensure passing a test class within a test file works fine without adding an extra "_test"?

Reply:
If the target ends with `_test`, it means a path to a test file has been provided. If it has `_test.` in it -- it means a path to a particular test or a method in a test file has been provided. If the path provided does not exist, error is raised when we try to execute the tests.